src/lib.rs

```diff
@@ -543,6 +544,43 @@ mod test {
         assert!(!MacAddr([1, 2, 3, 4, 5, 6]).is_unspecified());
     }
 
+    // Ensure that the `Display` and `Debug` traits are implemented on the `IpNetwork` struct
+    #[test]
+    fn check_display_impl_ip_network_ipv4() {
+        println!(
+            "{} {:?}",
+            IpNetwork::new(IpAddr::from(Ipv4Addr::new(1, 2, 3, 4)), 3),
+            IpNetwork::new(IpAddr::from(Ipv4Addr::new(255, 255, 255, 0)), 21)
+        );
+    }
+
+    #[test]
+    fn check_display_impl_ip_network_ipv6() {
+        println!(
+            "{} {:?}",
+            IpNetwork::new(
+                IpAddr::from(Ipv6Addr::new(0xffff, 0xaabb, 00, 0, 0, 0x000c, 11, 21)),
+                127
+            ),
+            IpNetwork::new(
+                IpAddr::from(Ipv6Addr::new(0xffcc, 0, 0, 0xffcc, 0, 0xffff, 0, 0xccaa)),
+                120
+            )
+        )
+    }
+
+    #[test]
+    fn check_ip_networks() {
+        if !IS_SUPPORTED_SYSTEM {
+            return;
+        }
+        let networks = Networks::new_with_refreshed_list();
+        if networks.is_empty() || networks.iter().any(|(_, n)| !n.ip_networks().is_empty()) {
```

Comment:
Why checking `networks.is_empty()`? Isn't it an issue if no networks are present?

Reply:
I once encountered a situation where there was no network interface in  `vmware` windows virtual machine.Not checking `networks.is_empty()` is better for find problems.

Reply:
Do you mind removing this test to check if it fails in the CI please?

Reply:
I removed it.